sections/slideshow.liquid

```diff
@@ -27,6 +27,47 @@
 {%- endif -%}
 
 <slideshow-component class="slider-mobile-gutter{% if section.settings.layout == 'grid' %} page-width{% endif %}{% if section.settings.show_text_below %} mobile-text-below{% endif %}" role="region" aria-roledescription="{{ 'sections.slideshow.carousel' | t }}" aria-label="{{ section.settings.accessibility_info | escape }}">
+  {%- if section.settings.auto_rotate and section.blocks.size > 1 -%}
+    <div class="slideshow__controls slideshow__controls--top slider-buttons no-js-hidden{% if section.settings.show_text_below %} slideshow__controls--border{% endif %}">
```

Comment:
>I feel like this PR could be simplified by adding a condition to include the new class to the existing control container:

```html
<div class="slideshow__controls {% if section.settings.auto_rotate %}slideshow__controls--top{% endif %} slider-buttons no-js-hidden{% if section.settings.show_text_below %} slideshow__controls--border{% endif %}">
```

@svinkle what do you mean when you say that ? 🤔 

To avoid the code duplication ? Cause if I kept the original controls and only changed the flexbox order it would not impact the tab navigation 🤔